excise/utils.py

```diff
@@ -204,14 +196,19 @@ def append_line_to_data(
     ).first()
     warehouse_address = stock.warehouse.address if stock else None
 
-    unit_of_measure = variant.product.product_type.\
-        get_value_from_private_metadata(get_metadata_key("UnitOfMeasure"))
+    unit_of_measure = (
+        variant.product.product_type.get_value_from_private_metadata(
+            get_metadata_key("UnitOfMeasure")
+        )
+    )
     unit_quantity = variant.get_value_from_private_metadata(
-        get_metadata_key("UnitQuantity"))
-    unit_quantity_of_measure = variant.product.product_type.\
-        get_value_from_private_metadata(
+        get_metadata_key("UnitQuantity")
+    )
+    unit_quantity_of_measure = (
+        variant.product.product_type.get_value_from_private_metadata(
```

Comment:
Potential optimization issue. We should be sure that `product_type` is prefetched.

Reply:
Here's the [PR (569ab01)](https://github.com/saleor/saleor-plugin-avatax-excise/pull/12/commits/569ab01)